### ISSUE TITLE
Someday/maybe

### ISSUE THREAD

**Issue by ADGEfficiency** [ADGEfficiency/energy_py#40]
A laundry list of potential future work:

- Cross Entropy Method
- Heuristic preprocessing - using rules to override agent policies
- Auxilary loss functions - state or reward predictions
- DQN extensions - prioritized experience replay, dueling etc.  See DeepMind Rainbow paper
- New gym envs - Lunar Lander

## Reward scaling

BonsAI have a great video on reward function design.

The techniques that would be relevant for energy_py would be scaling using the max reward (max = peak capacity * max price). The issue with this is that the reward is no longer directly interpretable as money.

This can be dealt with by including a cost in the env info dict.

Part of this work would be looking at how the distribution of rewards changes before and after scaling.
